runtime/src/qio/qio_formatted.c

```diff
@@ -1044,6 +1044,37 @@ char _qio_tohex(unsigned char i)
   else return 'A' + i - 10;
 }
 
+static
+int _qio_byte_escape(uint8_t b, int32_t string_end, int string_format, char* tmp, int *width_chars_out, int *width_cols_out)
+{
+  int i = 0;
+  char tmpchr;
+  int cwidth;
+  if( b == string_end || b == '\\' ||
+      b == '\'' || b == '"' || b == '\n' ) {
+    tmp[0] = '\\';
+    if( b == '\n' ) {
+      tmpchr = 'n';
+    } else {
+      tmpchr = b;
+    }
+    tmp[1] = tmpchr;
+    cwidth = 2;
+  } else if( !iswascii(b) || !isprint(b) ) {
```

Comment:
I've never heard of iswascii and my system doesn't have a `man` for it. Can you instead simply check `b >= 128`?